chapters/appendix-VI-external-repository-identifiers.md

```diff
@@ -14,59 +14,68 @@ There are currently four defined categories:
 The following sections provide details on the available types
 and the locator formats for each of the categories.
 
-
 ## Security
 
 ### cpe22Type <a name="cpe22"></a>
 
 Locator Format:
 
-    "[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
+```text
+"[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
```

Comment:
Should the surrounding quotation marks (") be removed from this? now that it is bordered by the backticks as well.